dev/get_schema.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Generate latest schema file for reference in development 
+# To be run from project root (so, './dev/get_schema.sh')
+#
+
+set -u
+set -e
+
+docker pull dockermediacloud/postgresql-server:master
+docker run dockermediacloud/postgresql-server:master cat /tmp/mediawords.sql > ./apps/postgresql-server/schema/mediawords.sql
```

Comment:
This is now `gcr.io/mcback`, not `dockermediacloud` anymore. Forgot to change it on my merge, sorry!

Also, I wouldn't pull anything here and would use the `latest` Docker tag instead of `master`. `latest` always [points (should point) to the last built / pulled image](https://github.com/mediacloud/backend/blob/master/doc/creating_apps.markdown#tag-versions), so if one is running this dev script, they're expecting to get the schema from the latest built / pulled image (be it `master`, `release` or some sort of a feature branch).

Reply:
Running into a funny problem when using `latest` tag rather than `master`:

```
jamesotoole@Jamess-MacBook-Pro-2 dev (jot-pgmigrate) $ ./get_schema.sh
Using default tag: latest
Error response from daemon: manifest for gcr.io/mcback/postgresql-server:latest not found: manifest unknown: Failed to fetch "latest" from request "/v2/mcback/postgresql-server/manifests/latest".
```
Confusing because I thought Docker automatically appends the latest tag to the most recently built image

Reply:
> Confusing because I thought Docker automatically appends the latest tag to the most recently built image

It does, but it doesn't (and shouldn't) push it to the Docker repository - `latest` is just, uh, a marker on your own machine pointing to the latest image that you yourself have pulled or built. Given that you don't (and shouldn't) `docker push ...:latest` to the repository, the `:latest` can't be pulled from it either.

Here's a bit more on that:

https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375

And again, our own doc:

https://github.com/mediacloud/backend/blob/master/doc/creating_apps.markdown#tag-versions

So, simply don't `docker pull` anything here (as you want to get the schema of whatever was last pulled / built locally), and use the `latest` tag instead of `master`, `release` or anything else.